cmd/cdi-importer/importer.go

```diff
@@ -88,6 +93,7 @@ func main() {
 		CertDir:            certDir,
 		InsecureTLS:        insecureTLS,
 		ScratchDataDir:     common.ScratchDataDir,
+		VolumeMode:         v1.PersistentVolumeMode(volumeMode),
```

Comment:
Since you managed to get volumeMode typed already, no need for this cast at all now.